move/operators/sources/operators.move

```diff
@@ -18,10 +20,46 @@ module operators::operators {
     /// The main `Operators` struct storing the capabilities and operator IDs.
     public struct Operators has key {
         id: UID,
-        operator_ids: vector<ID>,
+        // number of operators should small and under the Sui vector limits
+        operators: VecSet<ID>,
```

Comment:
```suggestion
        operators: VecSet<address>,
```
missing change to address instead of ID

Reply:
I though we agreed on the call to use Capabilities instead of the relaying on the address of TX origin. If we store the operator address instead how are we going to validate the Capability? Because removing the address from the list still allows to transfer the capability to another address and use it.

Reply:
But that address won't likely to be whitelisted. Ok I will make the change